### DISCUSSION TITLE
Add support to 32 bit platform.

### DISCUSSION THREAD

**Discussion by yangu-hury** [denoland/deno#10727] [General]
Deno should have support to x86 of 32 bit. There is a lot of peoples that use this arch until today. Like me :). That is a good for compatibility and less memory usage.

**Comment by lucacasonato** [comment]
Deno will not add support for 32 bit architectures. There are too few people that can take advantage of it compared to the very high difficulty and maintenance cost of implementation. 32 bit arch are dead end platforms. Also see https://github.com/denoland/deno/issues/3438#issuecomment-678697954.
